### PR TITLE
fix(onboarding-1.5): Small UI fixes

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

- fix sidebar layout on mobile
- fix input persistence bug

before:
https://user-images.githubusercontent.com/25164963/172061564-ef2b7170-d280-4e86-8c47-c9fdb380ae78.mov

after:
https://user-images.githubusercontent.com/25164963/172061565-d95c7818-ae9b-4e4f-b741-374cf137aa61.mov

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/ingestion/IngestionWizard.tsx [hunk 1]
```diff
@@ -90,6 +91,8 @@ function IngestionContainer({ children }: { children: React.ReactNode }): JSX.El
     const { isInviteModalShown } = useValues(inviteLogic)
     const { hideInviteModal } = useActions(inviteLogic)
     const { onboardingSidebarEnabled } = useValues(ingestionLogic)
+    const screens = useBreakpoint()
+    const isSmallScreen = screens.xs || (screens.sm && !screens.md)
```
Comment (alexkim205) [inline frontend/src/scenes/ingestion/IngestionWizard.tsx:95]: Same here

File: frontend/src/scenes/organization/Settings/InviteModal.tsx [hunk 2]
```diff
@@ -84,10 +83,10 @@ function InviteRow({ index, isDeletable }: { index: number; isDeletable: boolean
                         Submit
                     </LemonButton>
                 ) : (
-                    <Input
+                    <LemonInput
```
Comment (alexkim205) [inline frontend/src/scenes/organization/Settings/InviteModal.tsx:86]: nice!

**Review by alexkim205 — DISMISSED** [review]
left some inline comments!

File: frontend/src/scenes/ingestion/CardContainer.tsx [hunk 3]
```diff
@@ -18,6 +19,8 @@ export function CardContainer({
     showFooter?: boolean
 }): JSX.Element {
     const { onboardingSidebarEnabled } = useValues(ingestionLogic)
+    const screens = useBreakpoint()
+    const isSmallScreen = screens.xs || (screens.sm && !screens.md)
```
[the remark this answers is not in the document]
Comment (liyiy) [reply frontend/src/scenes/ingestion/CardContainer.tsx:23]: can you link me to an example of this in our code or the official docs I can't find anything

[the remark this answers is not in the document]
Comment (alexkim205) [reply frontend/src/scenes/ingestion/CardContainer.tsx:23]: 👍 here are the docs - https://keajs.org/docs/plugins/window-values/

**Review by alexkim205 — APPROVED** [review]

**Pull request merged**